apps/dashboard/src/app/(dashboard)/article/write-form.tsx

```diff
@@ -0,0 +1,60 @@
+import { type ArticleWrite, ArticleWriteSchema } from "@dotkomonline/types"
+import { createTextareaInput, createTextInput, useFormBuilder } from "../../form"
+
+const ARTICLE_FORM_DEFAULT_VALUES: Partial<ArticleWrite> = {}
+
+interface UseArticleWriteFormProps {
+  onSubmit(data: ArticleWrite): void
+  defaultValues?: Partial<ArticleWrite>
+  label?: string
+}
+
+export const useArticleWriteForm = ({
+  onSubmit,
+  label = "Legg inn ny artikkel",
+  defaultValues = ARTICLE_FORM_DEFAULT_VALUES,
+}: UseArticleWriteFormProps) =>
+  useFormBuilder({
+    schema: ArticleWriteSchema,
+    defaultValues,
+    onSubmit,
+    label,
+    fields: {
+      title: createTextInput({
+        label: "Tittel",
+        placeholder: "Fadderuka 2023",
+        withAsterisk: true,
+      }),
+      author: createTextInput({
+        label: "Forfattere",
+        placeholder: "Ola Nordmann, Trond-Viggo Torgersen",
```

Comment:
I would like a select component for this

Reply:
Hard to do with the current state of ow_user would have to search for their ulids right now.

Reply:
Should we sync up the attributes of users in cognito with ow_user maybe? It seems very unfortunate to not be able to do stuff like this. However, I agree that it does not need to be a part of this issue, so text input is fine!